csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/IRConstruction.qll

```diff
@@ -17,7 +17,6 @@ TranslatedElement getInstructionTranslatedElement(Instruction instruction) {
 }
 
 InstructionTag getInstructionTag(Instruction instruction) { instruction = MkInstruction(_, result) }
-
 import Cached
```

Comment:
It looks like you are using an old autoformatter; the latest autoformatter inserts a linebreak here.

Reply:
Everything should be fine now.